### ISSUE TITLE
Add request header If-Unmodified-Since

### ISSUE THREAD

**Issue by cmeeren** [relentless/Http.fs#168]
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Unmodified-Since

**Comment by ivpadim** [comment]
@cmeeren the api allows you to set a custom header in your request. Have you tried?

**Comment by cmeeren** [comment]
Yes, that works. Ideally I'd like to set a `DateTimeOffset` and have the header value formatted automatically. Currently I have to use `ToString("R")` manually. It's not too bad, of course, but an API accepting `DateTimeOffset` directly would be better. :)
